src/views/DatasetManager/DatasetManager.js

```diff
@@ -2,16 +2,19 @@
 // Licensed under the MIT license.
 
 import React from 'react';
-import { DatasetBrowser } from './components';
 import { Grid } from '@mui/material';
+import { DatasetBrowser, DatasetOverview } from './components';
 
 const DatasetManager = () => {
   return (
     <div data-cy="dataset-manager-view">
-      <Grid container>
-        <Grid item>
+      <Grid container spacing={0} sx={{ alignItems: 'stretch', justifyContent: 'flex-start' }}>
+        <Grid item xs={12} sm={4} md={3}>
```

Comment:
It's a very subjective point of view, but I'd suggest to remove the `sm` breakpoint because the screen is rather crowded and horizontal layout for 600px of width is too heavy. I'd go for` xs={12} md={4}` for _DatasetBrowser_ part and _xs={12} md={8}_ for overview part :slightly_smiling_face:

Reply:
Done, thank you for the suggestion :slightly_smiling_face: